eZ/Publish/Core/Repository/Helper/UserPasswordValidator.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository\Helper;
```

Comment:
I don't see a reason for exposing this here, esp. that this is internal.
```suggestion
namespace eZ\Publish\Core\FieldType\User\Validator;
```

Reply:
@adamwojs I still see this helper :P

Reply:
As discussed on slack: UserPasswordValidator has been moved into `eZ\Publish\Core\Repository\Validator` namespace.